docs/how-to-guides/block-tutorial/extending-the-query-loop-block.md

```diff
@@ -0,0 +1,195 @@
+# Extending the Query Loop block
+
+The Query Loop block is a powerful tool that allows users to cycle through a determined list of posts and display a certain set of blocks that will inherit the context of each of the posts in the list. For example, it can be set to cycle through all the posts of a certain category and for each of those posts display their featured image. And much more, of course!
+
+But precisely because the Query Loop block is so powerful and allows for great customization, it can also be daunting. Most users wouldn't want to be presented with the full capabilities of the Query Loop block, as most users wouldn't be familiar with the concept of a “query” and its associated technical terms. Instead, most users will likely appreciate a pre-set version of the block, with fewer settings to adjust and clearer naming. The Post List variation offered by default is a good example of this practice: the user will be using the Query Loop block without being exposed to its technicalities, and will also be more likely to discover and understand the purpose of the block.
+
+In the same manner, a lot of extenders might need a way to present bespoke versions of the block, with their own presets, additional settings and without customization options which are irrelevant to their use-case (often, for example, their custom post type). The Query Loop block offers very powerful ways to create such variations.
+
+## Extending the block with variations
+
+By registering your own block variation with some specific Query Loop block settings, you can have finer control over how it is presented, while still being able to use the full capabilities which the Query Loop block offers underneath. If you are not familiar with block variations, learn more about them [here](/docs/reference-guides/block-api/block-variations.md).
+
+With the block variations API you can provide the default settings that make the most sense for your use-case.
+
+Let's go on a journey, for example, of setting up a variation that could fit a plugin which registers a `book` [custom post type](https://developer.wordpress.org/plugins/post-types/).
```

Comment:
The thing is that the example is spanning through the entire document, so I don't feel like it makes sense to scope it under a section heading.

As for the rewording, do you have suggestions? Or, what's the feeling that you get from this paragraph that puts you off? Is it the phrasing, or the syntax? Let me know so I can provide a suggestion for the rewording. Or feel free to change it!

Reply:
No strong opinion on the heading.. Regarding the rewording the only thing that felt 'weird' to me was: `that could fit a plugin`. Specifically the `fit`. I'm not sure if it's just me, as I'm not a native English speaker..

Maybe it can be as simple as changing `that could fit -> for`.

Reply:
You're right. Maybe "suit" was a better choice than "fit". But I have changed it with "for" to keep it simple